editor/find_in_files.h

```diff
@@ -233,3 +240,45 @@ class FindInFilesPanel : public Control {
 	LineEdit *_replace_line_edit = nullptr;
 	Button *_replace_all_button = nullptr;
 };
+
+class PopupMenu;
+class TabContainer;
+
+// Contains several panels
```

Comment:
```suggestion
// Contains several panels.
```
Also would it be possible to make this more specific?

Reply:
Done. More details added.